### DISCUSSION TITLE
Menus are different in different contexts (start page is not Overview page)?

### DISCUSSION THREAD

**Discussion by testor2897** [pyinstaller/pyinstaller#7606] [Help]
Hello,

if I start the github page with a link from 

> https://pyinstaller.org/en/stable/

I am routed to:

> https://github.com/pyinstaller/pyinstaller/blob/5c9f3e65affd43faa087da3ef6ddea6fa5947f98/doc/index.rst

On this page there are several menus:

> [Code](https://github.com/pyinstaller/pyinstaller)
Issues
Pull requests
[Discussions](https://github.com/pyinstaller/pyinstaller/discussions)
[Actions](https://github.com/pyinstaller/pyinstaller/actions)
Projects
[Wiki](https://github.com/pyinstaller/pyinstaller/wiki)
Security
[Insights](https://github.com/pyinstaller/pyinstaller/pulse)

If I enter a menu (i.e. Discussion) and go back to Overview  some menus are "lost" and the remaining menus are:

> [Overview](https://github.com/pyinstaller)
> Repositories
> [Discussions](https://github.com/orgs/pyinstaller/discussions)
> [Projects](https://github.com/orgs/pyinstaller/projects)
> [Packages](https://github.com/orgs/pyinstaller/packages)
> People

So I miss at least Wiki and Pull requests.

Why is Overview a different page than the start page? Is it possible to map the Overview to the original start page?

**Comment by bwoodsend** [comment]
This is purely GitHub's web design - not a decision we've made. The reason why discussions has less menu options is because discussions are an organization wide whereas the first page is specific to one repository. i.e. https://github.com/orgs/pyinstaller/ instead of https://github.com/pyinstaller/pyinstaller/
